### PR TITLE
Add instructions for other systems

### PR DESCRIPTION
**Pull request by kaplan-michael**
Add installation and setup instructions for MacOS and Windows

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,27 +2,42 @@
...
 A library and command line interface for the CVE Project services.
 
+## Reequirements
+
+- Python version 3.6 or greater
+- pip
+
 ## Installation
 
+Universal Installation that should work on Linux, MacOS and Windows.
```
Comment (mprpic) [inline README.md:12]: Is this true for Windows? Looking at https://packaging.python.org/tutorials/installing-packages/#installing-from-pypi, the instructions include `py -m` before the `pip install`. Maybe we could link to this URL below the `pip install` command with something like: 

> For more information on installing Python packages from PyPI, see the [Python Packaging User Guide](https://packaging.python.org/tutorials/installing-packages/#installing-from-pypi).

File: README.md [hunk 2]
```diff
@@ -2,27 +2,42 @@
...
 
+### Windows
+
+```
+set CVE_USER=margo
+set CVE_ORG=acme
+set CVE_API_KEY=<api_key>
+```
+
 Additionial options that have an accompanying environment variable include:
```
Comment (mprpic) [inline README.md:41]: Unrelated, but might as well fix:

```suggestion
Additional options that have an accompanying environment variable include:
```

:-)

File: README.md (hunk 1 above)
Comment (kaplan-michael) [reply README.md:12]: You have to have pip installed on windows, then it should be the case. If you don't, you would do it as you say.

**Comment by kaplan-michael** [comment]
Fixed the typos and added the link to official packaging docs, Thanks

**Review by mprpic — APPROVED** [review]

**Pull request merged**